spec/models/vm_or_template_spec.rb

```diff
@@ -102,6 +102,20 @@
     end
   end
 
+  describe "save_genealogy_information" do
+    let(:vm) { FactoryBot.create(:vm_vmware) }
+    let(:parent) { FactoryBot.create(:vm_vmware) }
+
+    it "sets parent" do
+      vm.with_relationship_type('genealogy') { vm.genealogy_parent = parent }
```

Comment:
```suggestion
      vm.genealogy_parent = parent
```

Reply:
can you also load a new vm after the save (the `@` variables make the test a little more peculiar)

```ruby
vm = Vm.find(vm.id)
```